ui-participant/src/login/RedirectFromOAuth.tsx

```diff
@@ -74,8 +74,8 @@ export const RedirectFromOAuth = () => {
               })
               const hubUpdate: HubUpdate = {
                 message: {
-                  title: 'Welcome to the study.',
-                  detail: 'Please read and sign the consent form below to complete registration.',
+                  title: `Welcome to ${portalStudy.study.name}`,
```

Comment:
Just curious since the JS is not strong, how do things work if `portalStudy` is null?

Reply:
if portal study (or study) were null here, an exception would be thrown and the user would probably get stuck with an eternal loading spinner.  Fortunately this is where typescript helps us confirm that that won't happen -- the `if` on line 70 checks for null.  If that `if` weren't there, (or if study was a nullable property on portalStudy), then typescript would refuse to allow this line